src/Stage/Emit/JavaScript.elm

```diff
@@ -128,6 +128,9 @@ emitExpr located =
             in
             "{" ++ bindingsJS ++ "}"
 
+        Case e branches ->
+            "TODO"
```

Comment:
Do you want to figure out the emit phase in the same PR? We could talk through it here / on the Discord.